build.gradle

```diff
@@ -315,51 +329,51 @@ dependencies {
-  testCompile group: 'info.solidsoft.gradle.pitest', name: 'gradle-pitest-plugin', version: versions.gradlePitest
-  testCompile group: 'org.codehaus.sonar-plugins', name: 'sonar-pitest-plugin', version: versions.sonarPitest
+  testImplementation group: 'org.pitest', name: 'pitest', version: versions.pitest
+  testImplementation group: 'info.solidsoft.gradle.pitest', name: 'gradle-pitest-plugin', version: versions.gradlePitest
+  testImplementation group: 'org.codehaus.sonar-plugins', name: 'sonar-pitest-plugin', version: versions.sonarPitest
 
-  testCompile group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
-  testCompile group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
-  testCompile group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
-  testCompile group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
-  testCompile "org.junit.vintage:junit-vintage-engine:${versions.junit}"
-  testCompile 'com.github.hmcts:fortify-client:1.2.1:all'
+  testImplementation group: 'net.serenity-bdd', name: 'serenity-core', version: versions.serenity
+  testImplementation group: 'net.serenity-bdd', name: 'serenity-junit', version: versions.serenity
+  testImplementation group: 'net.serenity-bdd', name: 'serenity-rest-assured', version: versions.serenity
+  testImplementation group: 'net.serenity-bdd', name: 'serenity-spring', version: versions.serenity
+  testImplementation "org.junit.vintage:junit-vintage-engine:${versions.junit}"
+  testImplementation 'com.github.hmcts:fortify-client:1.2.1:all'
 
   //Pact contract testing
-  contractTestCompile group: 'au.com.dius.pact.consumer', name: 'junit5', version: versions.pactVersion
-  contractTestCompile group: 'au.com.dius.pact.consumer', name: 'java8', version: versions.pactVersion
+  contractTestImplementation group: 'au.com.dius.pact.consumer', name: 'junit5', version: versions.pactVersion
+  contractTestImplementation group: 'au.com.dius.pact.consumer', name: 'java8', version: versions.pactVersion
 
-  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5', version: versions.pactVersion
-  contractTestCompile group: 'au.com.dius.pact.provider', name: 'spring', version: versions.pactVersion
-  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5spring', version: versions.pactVersion
+  contractTestImplementation group: 'au.com.dius.pact.provider', name: 'junit5', version: versions.pactVersion
+  contractTestImplementation group: 'au.com.dius.pact.provider', name: 'spring', version: versions.pactVersion
+  contractTestImplementation group: 'au.com.dius.pact.provider', name: 'junit5spring', version: versions.pactVersion
 
-  contractTestCompile("org.junit.jupiter:junit-jupiter-api:5.6.2")
-  contractTestRuntime("org.junit.jupiter:junit-jupiter-engine:5.6.2")
+  contractTestImplementation("org.junit.jupiter:junit-jupiter-api:5.6.2")
```

Comment:
why is this version different to `versions.junit`

Reply:
I've not changed the version, only changed contractTestCompile to contractTestImplementation